### DISCUSSION TITLE
2022/08/20/extract-emotestamp-from-nintendo-switch-sports/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#11] [Announcements]
# 2022/08/20/extract-emotestamp-from-nintendo-switch-sports/index

SumyGG's Blog

https://sumygg.com/2022/08/20/extract-emotestamp-from-nintendo-switch-sports/index.html

**Comment by jesuxel** [comment]
可以分享一下吗？
